vault_s2i/Dockerfile

```diff
@@ -4,6 +4,7 @@ COPY . /
 
 ARG VAULT_GUEST_PORT
 RUN apk add --no-cache python3 && \
+    pip3 install --upgrade pip && \
```

Comment:
I'm not sure I follow the relation to the topic of the PR, and there is no description provided...
What's the issue it aims to solve?

Reply:
+1
Added in the PR description